ansible/roles/cas5/templates/application.yml

```diff
@@ -228,7 +228,7 @@ cas:
 ala:
   userDetailsBaseUrl: {{ userdetails_base_url }}/{{ userdetails_context_path | default('userdetails') }}/
   cookie:
-    domain: {{ auth_cookie_domain | default('ala.org.au') }}
+    domain: {{ cas_auth_cookie_domain | default(auth_cookie_domain) | default('ala.org.au') }}
```

Comment:
This is different than the one defined in userdetails that is `.ala.org.au` and cas fails to boot if it uses `.ala.org.au`. As we cannot use the default values, we need to have different variables to properly configure them.